entry.go

```diff
@@ -95,7 +95,7 @@ func (entry Entry) log(level Level, msg string) {
 
 	if err := entry.Logger.Hooks.Fire(level, &entry); err != nil {
 		entry.Logger.mu.Lock()
```

Comment:
I believe the reason is that since the default for `logrus` is to log to `stderr` this could mix into the logging otherwise and cause corrupted, strange output.